docs/docs-source/docs/modules/run-cloudflow-gcp-marketplace/pages/run-sample-application.adoc

```diff
@@ -0,0 +1,102 @@
+
+Next we'll deploy a sample Cloudflow application called https://github.com/lightbend/cloudflow/tree/master/examples/call-record-aggregator[call-record-aggregator] to the cluster.
+Clone the Cloudflow https://github.com/lightbend/cloudflow[Github repo] and `cd` into the `examples/call-record-aggregator` directory. Then run the following command to build the application:
+
+[source,bash]
+----
+$ sbt buildApp
+----
+
+Once it's complete, you should see at the end of the output a `kubectl cloudflow deploy` command to deploy your application. Run it:
+
+[source,bash]
+----
+$ kubectl cloudflow deploy call-record-aggregator.json
+----
+
+Once the deployment of the application starts, you can view the progress of the application using the `kubectl cloudflow status` command:
+
+[source,bash]
+----
+$ kubectl cloudflow status call-record-aggregator
+Name:             call-record-aggregator
+Namespace:        call-record-aggregator
+Version:          473-d654162-dirty
+Created:          2020-06-23 10:17:00 -0400 EDT
+Status:           Running
+
+STREAMLET         POD                                                        READY             STATUS            RESTARTS
+cdr-aggregator    call-record-aggregator-cdr-aggregator-driver               1/1               Running           0
+cdr-aggregator    call-record-aggregator-cdr-aggregator-1592921825294-exec-1 1/1               Running           0
+cdr-generator1    call-record-aggregator-cdr-generator1-driver               1/1               Running           0
+cdr-generator1    call-record-aggregator-cdr-generator1-1592921825612-exec-1 1/1               Running           0
+cdr-generator2    call-record-aggregator-cdr-generator2-driver               1/1               Running           0
+cdr-generator2    call-record-aggregator-cdr-generator2-1592921825136-exec-1 1/1               Running           0
+cdr-ingress       call-record-aggregator-cdr-ingress-665b79f7b5-b8k6g        1/1               Running           0
+console-egress    call-record-aggregator-console-egress-dd8667b84-lv7tm      1/1               Running           0
+error-egress      call-record-aggregator-error-egress-87574b5c4-w7mqs        1/1               Running           0
+split             call-record-aggregator-split-54f5844cf5-z8fp2              1/1               Running           0
+----
+
+== Using Lightbend Console
+Lightbend Console is a suite of tools provided by Lightbend that enables observing and monitoring Kubernetes applications.
+To monitor the call-record-aggregator application you just deployed, first find the pod in the namespace where you deployed Cloudflow (by default the `default` namespace) whose name starts with `console-frontend-`:
+
+[source,bash]
+----
+$ kubectl -n <cloudflow-namespace> get pods | awk '{print $1}' | grep -e "console-frontend"
+----
+
+Then run:
+[source,bash]
+----
+$ kubectl -n <cloudflow-namespace> port-forward pod/<console-frontend-pod> 8080:8080
```

Comment:
To skip a step, as there's a deployment for `console-frontend`, they can just run:

```
 kubectl port-forward deploy/console-frontend 8080
```

Reply:
I actually didn't know that port-forwarding works on deployment. Sure this is simpler. I'll update this part.